include/common_structs.h

```diff
@@ -564,12 +626,16 @@ typedef struct Entity {
     /* 0x3C */ UNK_PTR renderSetupFunc; // pointer to draw func(?)
     /* 0x40 */ union {
         s32* any;
+        SwitchData* swtch;
```

Comment:
unfortunate that we have to do this since `switch` is a keyword..wondering if there's a better alternative but yeah mawp

Reply:
thought about switchData, but none of the other union members have Data as a suffix. could just spell switch wrong like the typical java class/clazz thing, but i think its unclear to use swetch/swotch/whatever as opposed to just dropping the vowel.

Reply:
yeah I agree. zwitch is possible too but kinda goofy and not ideal because it starts with a different letter